packages/administration/assets/src/styles/main.scss

```
@@ -22,6 +22,7 @@
 @import 'toc';
 @import 'tom-select';
 @import 'category-tree';
+@import 'pinned-menu';
```

Stylelint (17.7.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)